pandas/core/internals.py

```diff
@@ -210,19 +210,18 @@ def make_block(self, values, placement=None, ndim=None, **kwargs):
 
         return make_block(values, placement=placement, ndim=ndim, **kwargs)
 
-    def make_block_scalar(self, values, **kwargs):
+    def make_block_scalar(self, values):
         """
         Create a ScalarBlock
         """
         return ScalarBlock(values)
 
-    def make_block_same_class(self, values, placement=None, fastpath=True,
-                              **kwargs):
+    def make_block_same_class(self, values, placement=None, **kwargs):
         """ Wrap given values in a block of same type as self. """
         if placement is None:
             placement = self.mgr_locs
         return make_block(values, placement=placement, klass=self.__class__,
-                          fastpath=fastpath, **kwargs)
+                          **kwargs)
```

Comment:
kwargs?

Reply:
The last pass pulled out the kwargs for `make_block` method; this is for make_block_same_class.  I'll do another pass and see how it goes.